### DISCUSSION TITLE
[Visual Studio] "ignore spaces in declaration statements" does not work when using "Format Document"

### DISCUSSION THREAD

**Discussion by valkyrienyanko** [Valks-Games/sankari#134] [General]
![image](https://user-images.githubusercontent.com/6277739/190888609-e512443c-8792-430e-9ba8-0a69359af1e2.png)

All the beautiful formatting on this would get reset on `Ctrl + K + D` (format document)

I'm not the only one who is having this problem 
https://stackoverflow.com/questions/26506563/visual-studio-2010-c-sharp-ignore-spaces-in-declaration-statements-not-wor

**Comment by policiu** [comment]
I vote to just let the formatter do its thing and not try to force it to keep the spaces.

**Reply by valkyrienyanko** [reply]
I'm just not going to use the format document hotkey
